builder/main.py

```
@@ -1595,6 +1595,65 @@ def download_fs_action(target, source, env):
         return 1
 
 
+def esp32_create_combined_bin(source, target, env):
```

Ruff (0.15.2)

[warning] Unused function argument: `source`

(ARG001)